test/plots/traffic-horizon.js

```diff
@@ -0,0 +1,31 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function() {
+  const data = await d3.csv("data/traffic.csv", d3.autoType);
+  const bands = 7;
+  const step = +(d3.max(data, d => d.value) / bands).toPrecision(2);
```

Comment:
You could use d3.tickStep here (or d3.tickIncrement if you want to be really fancy).

Reply:
Not sure how? With 7 bands and a maximum value = 9616, the step is rounded to 1400, but d3.ticks(0, max, bands + 1) returns 9 bands (i.e. 10 ticks = [0, 1000, 2000, …, 9000]).

Besides the technical point, I like the original example which clearly shows (with the select for the number of bands) how horizon charts are built.

Reply:
In my PR #753 you can say:

```js
const bands = 7; // just a hint; not guaranteed
const max = d3.max(data, d => d.value);
const step = d3.tickStep(0, max, bands);
const ticks = d3.ticks(0, max, bands);
```

This results in 10 bands spaced every 1,000:

<img width="940" alt="Screen Shot 2022-02-10 at 10 39 21 AM" src="https://user-images.githubusercontent.com/230541/153474565-e939a799-d69f-4eaa-bf8b-41cf89626a71.png">

Reply:
Or asking for five bands and labeling the lower bound rather than upper bound:

<img width="935" alt="Screen Shot 2022-02-10 at 10 41 55 AM" src="https://user-images.githubusercontent.com/230541/153474968-df595445-cfa3-4ab1-b228-dcaa11f9c0c6.png">